drivers/ptx/src/main/java/uk/ac/manchester/tornado/drivers/ptx/graal/nodes/vector/VectorStoreGlobalMemory.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, APT Group, Department of Computer Science,
+ * School of Engineering, The University of Manchester. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
```

Comment:
This is not the correct license header. Please take a look at:

https://github.com/beehive-lab/TornadoVM/blob/master/drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/phases/TornadoIfCanonicalization.java 

That style is the header + update to 2021.

Reply:
Sorry, but I don't see the difference between the two headers. What exactly is it ?

Reply:
Since we extend a class from Graal we need this:

```
 * Copyright (c) 2009, 2017, Oracle and/or its affiliates. All rights reserved.
 * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
```